source/developer-guides/boot-flow.rst

```diff
@@ -36,6 +36,118 @@ Payload    Load, verify and launch OS images; or perform firmware update
   If verification fails, one of two things occurs. If resiliency is enabled, the boot is swapped over to the alternate partition
   to recover the broken partition (see :ref:`firmware-resiliency-and-recovery`). Otherwise, the boot is halted.
 
+Stage 1A
+~~~~~~~~
+
+When coming out of reset, the processor may be in a very raw state and needs some basic
+initialization before the rest of the platform can be initialized. Stage 1A sets up the
+initial exectution environment to support the execution of the following stages. At this
+stage, a stack is not available and hence assembly used to carry out the basic initialization
+steps. Stage 1A is also excuted from the flash (Read-only memory) and thus does not use
+global variables.
+
+For Intel x86 processors, the first few instructions for the reset vector code need to be
+at the predefined reset vector address for the CPU to begin execution. Stage 1A contains a
+module called VTF (Volume Top File) which is placed at the top within the Stage 1A FV.
+The VTF contains the reset vector code and hence the VTF needs to be placed at an appropriate
+address so that the reset vector matches the Vtf0 file (Identified by the GUID ``1BA0062E-C779-4582-8566-336AE8F78F09``).
+The entry point address is specified by the ``_ModuleEntryPoint`` symbol is placed as the first

```

Comment:
The entry point for the Stage1A module within the Stage1A FV (_ModuleEntryPoint)  is placed as tge furst DWORD...